packages/app/vite.config.mjs

```diff
@@ -29,5 +30,9 @@ export default makeConfig({
         dest: 'dist',
       }],
     }),
+    Legacy({
+      targets: ['Chrome >= 64', 'Firefox >= 86', 'Edge >= 79'],
+      modernPolyfills: true,
```

Comment:
Why enable this option? The docs seem to recommend against it. I think all we care about is the `polyfills` option which is `true` by default and auto-generated off the `targets` entries

Reply:
It didn't work for me without this option set 😕

Reply:
Huh, interesting. I must be misunderstanding the intent of that field.